### DISCUSSION TITLE
20

### DISCUSSION THREAD

**Discussion by jojoldu** [jojoldu/blog-comments#844] [Comments]
http://jojoldu.tistory.com/20

**Comment by jojoldu** [comment]
작성자: 재러리

공감가는 글이네요. 특히 마지막 ...

**Comment by MASKUN2** [comment]
뜬금없지만.. 너무 오래된 글을 다시보러 왔습니다. 
저도 "그동안 몰랐던 내용을 공부해서 적용하였는데 이걸 자랑할 사람이 없다면?

생각보다 빠른시간안에 높은 퀄리티로 개발완료 하게 됐는데 그걸 알아줄 사람이 없다면?



하루 혹은 일주일 동안 공부한 내용을 남들에게 얘기하고 칭찬받고 격려받고 자랑하는것. 

이게 너무 재미있고 더 열심히 하게 만드는 계기가 되는 것같다." 를 보고 많은 생각을 하게 되었어요. 

저는 신입 개발자고 아직 회사에 그런 분위기가 없거든요. 저희팀의 개발자도 저 포함 2명인데 한명은 프론트 개발자이고 저는 백엔드 개발자라서 그런 딮한 교류가 어렵습니다.

이럴 때면 나는 어떻게 해야할까 고민해봅니다. 
새로운 세계로 떠나거나, 아니면 내가 그것을 만들거나.
